### PR TITLE
adding histropedia page

### PR DESCRIPTION
**Pull request by rubyimrie**
adding histropedia page, no big changes

### CODE REVIEW

File: components/MainNavBar.vue [hunk 1]
```diff
@@ -96,7 +96,11 @@ export default {
               url: "/",
               external: false,
             },
-            { name: "Timeline Search", url: "/", external: false },
+            {
+              name: "Histropedia Timeline",
```
Comment (Sephster) [inline components/MainNavBar.vue]: I think most users won't know what Histropedia is so it is probably better to revert this to the original title and change the page filename accordingly. Thanks Ruby

File: pages/histropedia-timeline.vue [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<template>
+  
+  <div style="height: 100%; min-height: 500px;">
```
Comment (Sephster) [inline pages/histropedia-timeline.vue]: Are there equivalent tailwind calls we can use here?

**Review by Sephster — CHANGES_REQUESTED** [review]

File: components/MainNavBar.vue (hunk 1 above)
Comment (rubyimrie) [reply components/MainNavBar.vue]: done

File: pages/histropedia-timeline.vue (hunk 2 above)
Comment (rubyimrie) [reply pages/histropedia-timeline.vue]: done

File: components/MainNavBar.vue (hunk 1 above)
Comment (Sephster) [reply components/MainNavBar.vue]: Thank you

File: pages/histropedia-timeline.vue (hunk 2 above)
Comment (Sephster) [reply pages/histropedia-timeline.vue]: Thanks Ruby. Looks good to me!

**Review by Sephster — APPROVED** [review]

**Pull request merged**